src/components/dar_drafts_table/DarDraftTable.js

```diff
@@ -0,0 +1,254 @@
+
+//helper function to create table metadata for update date cell metadata
+const updateDateCell = ({updateDate, id, style = {}, label = 'draft-update-date'}) => {
+  return {
+    data: formatDate(updateDate),
+    id,
+    style,
+    label
+  };
+};
+
+//sub-component that redirects user to draft applictaion page
+const ResumeDraftButton = (props) => {
+  const { draft, history } = props;
+  const { referenceId } = draft;
+  return h(SimpleButton, {
+    keyProp: `resume-draft-${referenceId}`,
+    label: 'Resume',
+    baseColor: Theme.palette.secondary,
+    additionalStyle: {
+      width: '30%',
+      padding: '2%',
+      marginRight: '2%',
+      fontSize: '1.45rem'
+    },
+    onClick: () => history.push(`/dar_application/${referenceId}`)
+  });
+};
+
+//helper function to create id string for error messages
+const getIdentifier = ({id, data}) => {
+  return !isEmpty(data) ? (data.projectTitle || data.partialDarCode) : id;
+};
+
+//sub-component that renders draft delete button
+const DeleteDraftButton = (props) => {
+  const { targetDraft, showConfirmationModal } = props;
+  const { id, data } = targetDraft;
+  const identifier = getIdentifier({id, data});
+  return h(SimpleButton, {
+    keyProp: `delete-draft-${identifier}`,
+    label: 'Delete',
+    baseColor: Theme.palette.error,
+    additionalStyle: {
+      width: '30%',
+      padding: '2%',
+      fontSize: '1.45rem',
+    },
+    onClick: () => showConfirmationModal(targetDraft),
+  });
+};
+
+//helper function that formats table metadata and components for draft action buttons
+const actionCellData = ({
+  targetDraft,
+  showConfirmationModal,
+  history,
+}) => {
+  const { id } = targetDraft;
+  const deleteButtonTemplate = h(DeleteDraftButton, {
+    targetDraft,
+    showConfirmationModal
+  });
+  const resumeButtonTemplate = h(ResumeDraftButton, {
+    draft: targetDraft,
+    history,
+  });
+
+  return {
+    isComponent: true,
+    label: 'draft-buttons',
+    data: div(
+      {
+        style: {
+          display: 'flex',
+          justifyContent: 'left',
+        },
+        key: `draft-buttons-cell-${id}`,
+      },
+      [resumeButtonTemplate, deleteButtonTemplate]
+    ),
+  };
+};
+
+//helper function that foramts table metadata for cells row by row, returns an array of arrays
+const processDraftsRowData = ({visibleDrafts, showConfirmationModal, history}) => {
+  if(!isEmpty(visibleDrafts)) {
+    return visibleDrafts.map((draft) => {
+      const { id, data, createDate, updateDate } = draft;
+      const { projectTitle } = !isEmpty(data) ? data : {};
+      const partialDarCode = 'DRAFT_DAR_' + formatDate(createDate);
+      return [
+        partialDarCodeCell({partialDarCode, id}),
+        projectTitleCell({projectTitle, id}),
+        updateDateCell({updateDate: updateDate || createDate, id}),
+        actionCellData({targetDraft: draft, showConfirmationModal, history})
+      ];
+    });
+  }
+};
+
+export default function DarDraftTable(props) {
```

Comment:
I think this file was deleted while we worked on this feature branch and adding it back in here is unintentional